src/vos/vos_internal.h

```diff
@@ -804,6 +804,9 @@ enum vos_iter_opc {
 	IT_OPC_NEXT,
 };
 
+D_CASSERT((int)VOS_ITER_CB_ABORT > (int)IT_OPC_NEXT);
+D_CASSERT((int)VOS_ITER_CB_YIELD > (int)IT_OPC_NEXT);
```

Comment:
It is some confused to mix use "VOS_ITER_XXX" and "IT_OPC_XXX" in the iteration action. Can we redefine them together?